lib/message_sender.rb

```diff
@@ -0,0 +1,39 @@
+class MessageSender
+
+	attr_accessor :sms_message
+	attr_reader :order_time, :delivery_time
+
+	def set_order_time(time)
+		@order_time = time.to_s.slice(11,5)
+	end
+
+	def order_time
+		@order_time
+	end
+
+	def delivery_time
+		array = order_time.split(":")	
+		array[0] == "23" ? array[0] = "00" : array[0] = array[0].next 
+		array.insert(1,":")
+		array.join
+	end
+
+	def sms_message
+		@sms_message = "Thank you! Your order was placed and will be delivered before #{delivery_time}"
+	end
+		
+	def send_message
+		require 'rubygems'
+		require 'twilio-ruby'
+		# Get your Account Sid and Auth Token from twilio.com/user/account
+		account_sid = 'AC224c736fce467ca8cf9b8b3bfa73a034'
```

Comment:
all ids etc. should be in env variables and accessed by ENV['TWILLIO_ACCOUNT_SID'] statements